test/e2e/knative_serving_test.go

```diff
@@ -26,13 +26,13 @@ func TestKnativeServing(t *testing.T) {
 
 	t.Run("verify correct deployment shape", func(t *testing.T) {
 		// Check the desired scale of deployments in the knative serving namespace
-		for _, deployment := range []string{"activator", "controller", "autoscaler-hpa"} {
+		for _, deployment := range []string{"activator", "controller", "autoscaler-hpa", "domain-mapping"} {
```

Comment:
We can add `domainmapping-webhook` and `webhook` here.

Reply:
Autoscaler has no ha settings?

Reply:
Yeah, autoscaler has the HA setting. :sweat_smile: Let's add it.